README.md

```diff
@@ -3,72 +3,93 @@
+- [publishing-api](https://github.com/alphagov/publishing-api) - will provide workflow for all content published to GOV.UK - creating a new document, publishing it, etc.
+- [router-api](https://github.com/alphagov/router-api) - API for updating routes
+- [govuk-content-schemas](http://github.com/alphagov/govuk-content-schemas) - to run the tests
+- local transactions data - there is no UI or automated process for importing the source data for local transactions. The source data can be downloaded from [DirectGov](http://local.direct.gov.uk/Data/local_authority_service_details.CSV). It can then be imported using a rake task:
 
-The source data can be downloaded from [http://local.direct.gov.uk/Data/local_authority_service_details.CSV](DirectGov).
+    bundle exec rake local_transactions:import SOURCE=/path/to/local_authority_service_details.CSV
 
-They can be imported using a rake task:
+- [statsd](https://github.com/etsy/statsd/) - this application uses [statsd-ruby](http://rubygems.org/gems/statsd-ruby) to send metrics to statsd. If a statsd process isn't present on the server it won't matter as statsd-ruby sends metrics over UDP. If a statsd process is present then
+it'll send strings with the respective increment/decrement/gauge function to use.
+- [asset-manager](https://github.com/alphagov/asset-manager) - manages uploaded assets (images, PDFs, videos etc.). Publisher needs an OAuth bearer token in order to authenticate with Asset Manager. By default, this is loaded from the PUBLISHER_ASSET_MANAGER_BEARER_TOKEN environment variable in config/initializers/gds_api.rb.
+To obtain this bearer token, you should create an API user in the signonotron2 application. In the signonotron2 directory, to generate the bearer token you need, run:
 
-    bundle exec rake local_transactions:import SOURCE=/path/to/local_authority_service_details.CSV
+```
+rake api_clients:create[publisher,publisher@example.com,asset-manager,signin]
+```
 
-## Statsd
+### Running the application
 
-This application uses [statsd-ruby](http://rubygems.org/gems/statsd-ruby) to send metrics to
-[statsd](https://github.com/etsy/statsd/). If a statsd process isn't present on the server
-it won't matter as statsd-ruby sends metrics over UDP. If a statsd process is present then
-it'll send strings with the respective increment/decrement/gauge function to use.
+If you're just interested in running the Publisher locally, with a minimum of interaction
+with other apps, here's how.
 
-## Asset Manager
+##### Create a user
 
-If your models include media assets such as images and video, you will need to run the asset-manager
-app alongside publisher.
+This step is if you haven't [imported production data](https://github.gds/gds/development#8-import-production-data).
 
-See the [asset-manager](http://github.com/alphagov/asset-manager) project for app-specific setup 
-instructions.
+publisher$ rails console
```

Comment:
Put these in a multi-line code block but without the prefix, then people can copy and paste.